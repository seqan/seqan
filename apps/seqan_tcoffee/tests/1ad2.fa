>rk1_cyapa
EEQLYEPLEAIQLLKQISTAKFIETVEVHCRLGIDPKYNDQQLRATVTKAGKLEFRNDRTGIVHVLFGKA
NFSEEALLGNLKAVQEAVDRNRPSGVKGKYWKSIYITSTMSPSIEVDLPNGTGQTLRVAVITRGEKINEA
STAGADLVGAEELIDEIQNGRLDFDCLIATPDMMPQVAKLGRVLGPRGLMPSPKGGTVTFDLAQAINDF
>rl1_halvo
VDAVSRALDEAPGRNFRETVDLAVNLRDLDLNDPSKRVDESIVADETDFFVAEAGLMQDIGRYLGTVLGP
RGKMPTPLQPDDDVVETVNRMKNTVQLRSRDRRTFHTRVGADDMTPDEIAENIDVIVRRLEATLEKGPLN
IDSVYVKTTMGPSVEVLPSGTGQDTQIVVFATGETAAEDAADEVLGPDELEDFGDDTDAAKDL
>rl1_metja
VKEARELAKPRNFTQSFEFIATLKEIDMRKPENRIKTEVVAKAHDFFIAQADLMPLIGRYMGVILGPRGK
MPKPVPANANIKPLVERLKKTVVINTRDKPYFQVLVGNEKMTDEQIVDNIEAVLNVVAKKYEKGLLPHGR
GKEAKIAVIGTGDLAKQAEELGLTVIRKEEIEELGKNKRKLRKIYHIKDAYVKLTMGPAVKVK
>rl1_sulso
AIQNEWFLINQESMALAGRILGPALGPRGKFPTPLPNTADISEYINRFDKESLIEALKLALSTEYNVKRN
FTQSVEIILTFKGIDMKKGDLKLREIVPKRSVIVKTKDQPQVQVFIGTEDMKPEDLAENAIAVLNAIENK
AKVELPKQPSKAKRVLVVPSSEQLEYAKKASPKVVITREELQKLQGQKRPVKKLTNLRNIYVKTTMGKAV
KVK
