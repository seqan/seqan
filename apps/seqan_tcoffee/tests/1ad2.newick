(rk1_cyapa,(rl1_halvo,(rl1_metja,rl1_sulso)))
