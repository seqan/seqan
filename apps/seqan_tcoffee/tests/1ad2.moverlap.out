>rk1_cyapa
--------------------------------------------EEQLYEPLEAIQLLKQ
ISTAKFIETVEVHCRLGIDPKYNDQQLRATVT----------------------------
------------------------KAGKLEFRNDRTGIVHVLFGKANFSEEALLGNLKAV
QEAVDRNRPSGVKGKYWKSIYITSTMSPSIEVDLPNGTGQTLRVAVITRGEKINE--AST
AG-ADLVGAEELIDEIQNGRLDFDCL-----IATPDMMPQVAKLGRVLGPRGLMPSPKGG
TVTFDLAQAINDF
>rl1_halvo
VDAVSRALDEAPGRNFRETVDLAVNLRDLDLNDPSKRVDESIVADETDFFVAEA-GLMQD
I--GRYLGTVLGPRGKMPTPLQPDDDVVETVNRM--------------------------
------------------------KNTVQLRSRDRRT-FHTRVGADDMTPDEIAENIDVI
VRRLEATLEKGPLN--IDSVYVKTTMGPSVEV-LPSGTGQDTQIVVFATGE--TA--AED
AA-DEVLGPDELEDFGDDTDAAKDL-----------------------------------
-------------
>rl1_metja
---VKEARELAKPRNFTQSFEFIATLKEIDMRKPENRIKTEVVAKAHDFFIAQA-DLMPL
I--GRYMGVILGPRGKMPKPVPANANIKPLVERL--------------------------
------------------------KKTVVINTRDKPY-FQVLVGNEKMTDEQIVDNIEAV
LNVVAKKYEKG--------------------L-LPHGRGKEAKIAVIGTGD--LAKQAEE
LG-LTVIRKEEIEELGKNKRKLRKI-YHIKDAYVKLTMGPAVKVK---------------
-------------
>rl1_sulso
-------------------------------------------AIQNEWFLINQ-ESMAL
A--GRILGPALGPRGKFPTPLPNTADISEYINRFDKESLIEALKLALSTEYNVKRNFTQS
VEIILTFKGIDMKKGDLKLREIVPKRSVIVKTKDQPQ-VQVFIGTEDMKPEDLAENAIAV
LNAIENK-AKV--------------------E-LPKQPSKAKRVLVVPSSE--QLEYAKK
ASPKVVITREELQKLQGQKRPVKKL-TNLRNIYVKTTMGKAVKVK---------------
-------------
